docs/ADL2/master04.5-cadl_primitive_types.adoc

```diff
@@ -544,17 +682,36 @@ Soft terminology constraints are specified in ADL using keywords prior to the fo
         ...
     }
 --------
+====
 
 ==== Operational Binding Constraints
 
-The above sections describe 'source form' constraints, i.e. constraints expressed in terms of internal codes and value-sets. A further constraint possibility exists, for use at the point of operational template generation. As described in <<From Constraints to Concrete Codes in Data>>, the choice may be made that a specific operational template (OPT) should use external codes from the archetype bindings (such as from {snomed_ct}[SNOMED CT^], {who_icd}[ICD10^], etc) as the values of some or all coded nodes, rather than using the internal at-codes. 
+The above sections describe 'source form' constraints, i.e. constraints expressed in terms of internal codes and value-sets. A further constraint possibility exists, for use at the point of operational template generation. As described in <<From Constraints to Concrete Codes in Data>>, the choice may be made that a specific operational template (OPT) should use external codes from the archetype bindings (such as from {snomed_ct}[SNOMED CT^], {who_icd}[ICD10^], etc) as the values of some or all coded nodes, rather than using the internal at-codes.
 
-To express this choice, the OPT's `definition` section contains a modified version of the usual syntax `[at1]` or `[ac1]` in those nodes where an external term from the bindings is to be used. This takes the form `[acN@ttttt]` or `[atN@ttttt]` where `ttttt` is the namespace identifier of a binding in the `terminology` section of the archetype.
+To express this choice, the OPT's `definition` section contains a modified version of the usual syntax `[at0001]` (`[at1]`) or `[ac1]` in those nodes where an external term from the bindings is to be used. This takes the form `[acN@ttttt]` or `[atNNNN@ttttt]` (`[atN@ttttt]`) where `ttttt` is the namespace identifier of a binding in the `terminology` section of the archetype.
```

Comment:
Shouldn't the example codes for _id_ scenario (at1 and ac1) be incremented by 1 (at2 and ac2)? I know this is how it was in the text previously, but my understanding is it was wrong, because you couldn't have at1 (given that id1 is the root node, and numbers should not be reused -- or was that just a "suggestion" and not mandatory?).